.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
     -   id: check-pylint-normal
         name: Check pylint (except unit tests)
         types: [file, python]
-        exclude: ^tests/garage/.*$  # exclude unit tests
+        exclude: (^tests/garage/.*$|^benchmarks/)  # exclude unit tests
```

Comment:
if you are going to do this because you feel some rules shouldn't apply to the directory, please add a custom .pylintrc to the `benchmarks/` dir and a custom pylint job to pre-commit, as we already do for tests.